types/highcharts/v6/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "array-type": false,
+        "ban-types": false,
+        "export-just-namespace": false,
```

Comment:
This lint rule should be pretty easy to enable (and I'd like to see it happen, since this isn't on in the old version of the types) - in the offending file, just remove the wrapping `namespace` declaration, add `export` modifiers to all the former namespace members, and remove the `export = namespace` line. We do this just to remove unnecessary complexity from declaration files. Given that it's an almost 8MB file, I imagine even the smallest bit of clarification in what's actually exported is helpful!